src/test/crossFetchNode.test.ts

```diff
@@ -274,3 +274,110 @@ test('should not fail when arbitrary parameters are configured in fetchOptions',
   });
   expect(response).toEqual({});
 });
+
+test('throws on responses other than and 304 errors', async () => {
+  nock('https://localhost:3000')
+    .get(
+      `/search/shopper-search/v1/organizations/${config.parameters.organizationId}/product-search?siteId=${config.parameters.siteId}&q=sony`
+    )
+    .matchHeader('authorization', 'Bearer test-auth')
+    .reply(
+      400,
+      {content: 'not empty'},
+      {'content-type': 'application-json charset=UTF-8'}
+    );
+
+  const standardConfig: ClientConfigInit<TestConfigParameters> = {...config};
+  const client = new ShopperSearch(standardConfig);
```

Comment:
```suggestion
  const client = new ShopperSearch({...config});
```
If you're not doing anything with it, you can just pass it directly. 😉